data/ytemplates/bootstrap/value.custom_link.tpl.php

```diff
@@ -7,7 +7,7 @@
 $parameters = array(
     'media' => ($this->getElement('media') == 1),
     'mailto' => ($this->getElement('mailto') == 1),
-    'external' => ($this->getElement('extern') == 1),
+    'external' => ($this->getElement('external') == 1),
```

Comment:
Ich kann bestätigen, dass dieser fix erforderlich ist @skerbis, ansonsten wird der external Button in den yForm Feldern nicht angezeigt.

Reply:
hihi ... habe ich tatsächlich auch gerade gefixt. - Danke